packages/flutter_tools/lib/src/isolated/devfs_web.dart

```diff
@@ -101,7 +101,10 @@ class WebExpressionCompiler implements ExpressionCompiler {
   }
 
   @override
-  Future<bool> updateDependencies(Map<String, String> modules) async => true;
+  Future<void> initialize({bool soundNullSafety}) async {}
+
+  @override
+  Future<bool> updateDependencies(Map<String, ModuleInfo> modules) async => true;
```

Comment:
This code is not used in the monorepo and should be perfectly safe to roll.

Reply:
is this something the tool needs to implement? @grouma

Reply:
Nope. Since you guys use the Frontend Server your dependencies are always up to date and it is always initialized with the correct null safety support.